### DISCUSSION TITLE
EF Core 7 JSON columns

### DISCUSSION THREAD

**Discussion by rwth-aachen1** [LuckyPennySoftware/AutoMapper#4251] [NA]
The issue #4211 contains a problem I currently have.

**Comment by lbargaoanu** [comment]
Check [the execution plan](https://docs.automapper.org/en/latest/Understanding-your-mapping.html).

**Comment by ofthelit** [comment]
[Entity Framework Core](https://github.com/dotnet/efcore#entity-framework-core) bug report: https://github.com/dotnet/efcore/issues/30830
